conf/ocp_version/ocp-4.6-config.yaml

```
@@ -0,0 +1,12 @@
+---
+# Config file for nightly OCP 4.6
+RUN:
+  client_version: '4.6.0-0.nightly'
+DEPLOYMENT:
+  installer_version: "4.6.0-0.nightly"
+  terraform_version: "0.12.26"
+ENV_DATA:
+  vm_template: 'rhcos-46.82.202006190240-0-vmware.x86_64.ova'
+  # rhcos_ami: 'ami-06c85f9d106577272'
+  # If rhcos_ami is not specified the ID for aws upi deployments will be fetched from
+  # https://raw.githubusercontent.com/openshift/installer/release-4.5/data/data/rhcos.json
```

Not a big deal because it's a comment but this link should be for 4.6

```suggestion
  # https://raw.githubusercontent.com/openshift/installer/release-4.6/data/data/rhcos.json
```

---

Done